tests/shared/src/test/scala/scala/meta/tests/parsers/Scala213Suite.scala

```diff
@@ -128,6 +128,38 @@ class Scala213Suite extends ParseSuite {
     )(dialects.Scala213Source3)
   }
 
+  test("issue-2880") {
```

Comment:
is there a test which can demonstrate that something used to be parsed one way, but is now parsed differently, or rejected?